patterns/events-list-with-cta.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Title: Events - List with Call to Action
  * Slug: twentytwentyfive/events-list-with-cta
```

Comment:
Shall we rename this to `cta-event-links` so that we follow the naming convention with the other call to actions?